### PR TITLE
add PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by AndriusBartulis**
_(Please remove yourselves as reviewers if you don't have any input and are happy with whatever)_

This PR adds an organisation level PR template.

This is just a rough idea and a starting point, please comment your suggested improvements, or commit to this branch freely.

### CODE REVIEW

**Comment by jam3sn** [comment]
Maybe update them to H2 (`##`), and change `### Ticket link` to `## [DEV-]()` so it's partially ready

**Comment by tomheaton** [comment]
if we rename this and move it to a `PULL_REQUEST_TEMPLATE` dir, we can store multiple nicely

**Comment by tomheaton** [comment]
also add spacing between headers and content (md best practices)

**Comment by AndriusBartulis** [comment]
> if we rename this and move it to a `PULL_REQUEST_TEMPLATE` dir, we can store multiple nicely

Do we need multiple at this stage? maybe it would be worth to keep it really simple to start with and just have a default one?

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# [DEV-]()
+
+## Changes summary
+...
+
+## Risks
+...
+
+## Manual testing done
+...
```
Comment (jam3sn) [inline PULL_REQUEST_TEMPLATE.md]: Can we drop the `...` too, it's just something to delete otherwise

Comment (AndriusBartulis) [reply PULL_REQUEST_TEMPLATE.md]: dropped, not sure if should keep the spacing or not? left spacing as is, i.e. one line space, then a line for content, then a line after it before next heading.

Comment (jam3sn) [reply PULL_REQUEST_TEMPLATE.md]: Think it makes sense right?

Comment (tomheaton) [reply PULL_REQUEST_TEMPLATE.md]: space is good, follows best practices

**Review by jam3sn — APPROVED** [review]

**Review by tomheaton — APPROVED** [review]

**Review by timmw — APPROVED** [review]

**Review by stephen-horvath — APPROVED** [review]

**Review by rlweb — APPROVED** [review]

**Review by twhc20 — APPROVED** [review]

**Pull request merged**